[2023-03-29T18:51:50.848+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table __airflow_temporary_run_2023-03-29T18:51:48.922573+00:00__ [queued]>
[2023-03-29T18:51:50.856+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table __airflow_temporary_run_2023-03-29T18:51:48.922573+00:00__ [queued]>
[2023-03-29T18:51:50.857+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-03-29T18:51:50.858+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-03-29T18:51:50.859+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-03-29T18:51:50.869+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2022-01-01 00:00:00+00:00
[2023-03-29T18:51:50.874+0000] {standard_task_runner.py:55} INFO - Started process 17675 to run task
[2023-03-29T18:51:50.877+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', '__***_temporary_run_2023-03-29T18:51:48.922573+00:00__', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp8ojc_ng_']
[2023-03-29T18:51:50.880+0000] {standard_task_runner.py:83} INFO - Job 17: Subtask create_table
[2023-03-29T18:51:50.934+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table __airflow_temporary_run_2023-03-29T18:51:48.922573+00:00__ [running]> on host 9dcb783b1f3a
[2023-03-29T18:51:50.983+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=__***_temporary_run_2023-03-29T18:51:48.922573+00:00__
[2023-03-29T18:51:50.990+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-03-29T18:51:50.995+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            )   
        , parameters: None
[2023-03-29T18:51:51.000+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-03-29T18:51:51.007+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20220101T000000, start_date=20230329T185150, end_date=20230329T185151
[2023-03-29T18:51:51.050+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-03-29T18:51:51.073+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
